verilog/parser/verilog.y

```diff
@@ -5208,13 +5220,10 @@ module_parameter_port_list_item_last
   ;
 
 net_declaration
-  : net_type data_type_or_implicit_followed_by_id_and_dimensions_opt
-    trailing_assign_opt ',' net_variable_or_decl_assigns ';'
-    // TODO(fangism): unpack/repack $2..$5
-    { $$ = MakeTaggedNode(N::kNetDeclaration, $1, $2, $3, $4, $5, $6); }
-  | net_type data_type_or_implicit_followed_by_id_and_dimensions_opt
-    trailing_assign_opt ';'
-    { $$ = MakeTaggedNode(N::kNetDeclaration, $1, $2, $3, nullptr, nullptr, $4); }
+  : net_type net_variable_or_decl_assigns ';'
+    { $$ = MakeTaggedNode(N::kNetDeclaration, $1, $2, $3); }
```

Comment:
To make `kNetDeclaration` shaped the same in all cases, I'd insert a `nullptr` between `$1` and `$2` here.

Reply:
Sure, let's do that - added, thanks!